packages/core/src/browser/common-frontend-contribution.ts

```diff
@@ -23,6 +23,8 @@ import { KeybindingContribution, KeybindingRegistry } from './keybinding';
 import { FrontendApplication } from './frontend-application';
 import { FrontendApplicationContribution, OnWillStopAction } from './frontend-application-contribution';
 import { CommandContribution, CommandRegistry, Command } from '../common/command';
+import { CommonCommands } from './common-commands';
+export { CommonCommands };
```

Comment:
With this change, the breaking change notice can probably be removed from the changelog: external consumers shouldn't be affected.